pkg/k8sclient/nodewatcher.go

```diff
@@ -115,6 +118,8 @@ func (nw *NodeWatcher) parseNode(node *v1.Node, phase NodePhase) *Node {
 		CPUAllocatable:   cpuAllocQuantity.MilliValue(),
 		MemCapacityKb:    memCap / bytesToKb,
 		MemAllocatableKb: memAlloc / bytesToKb,
+		PodCapacity:      podCapQuantity.Value(),
```

Comment:
It's never really used by firmament? We just pass the `PodAllocatable` to firmament?

Reply:
@m1093782566  Only use `PodAllocatable` now. I'll check whether `PodCapacity` should be used.